include/swift/Demangling/Demangle.h

```diff
@@ -649,6 +649,11 @@ ManglingErrorOr<const char *> mangleNodeAsObjcCString(NodePointer node,
 std::string nodeToString(NodePointer Root,
                          const DemangleOptions &Options = DemangleOptions());
 
+/// Transforms a mangled key path accessor thunk helper
+/// into the identfier/subscript that would be used to invoke it in swift code.
+std::string keypathSourceString(const char *MangledName,
```

Comment:
I _think_, based on a cursory look through other C++ sources, that we still spell it `keyPath` (note the capitalization) in compiler-internal contexts.